sdk/azcore/internal/exported/request_test.go

```diff
@@ -81,9 +84,32 @@ func TestRequestBody(t *testing.T) {
 	if err := req.Close(); err != nil {
 		t.Fatal(err)
 	}
+	if req.Body() != nil {
+		t.Fatal("expected nil body")
+	}
```

Comment:
```suggestion
	require.NotNil(t, req.Body())
```

Reply:
I'm going to make a separate pass to clean these things up.